src/ext/JourneyPatternStopPointMap/JourneyPatternStopPointMap.tsx

```diff
@@ -10,10 +10,10 @@ import { getStopPlaces } from '../../actions/stopPlaces';
 import { useDispatch } from 'react-redux';
 import { StopPointLocation } from '../../reducers/stopPlaces';
 import { StopPlace } from '../../api';
-import QuaysWrapper from './QuaysWrapper';
+import QuaysWrapper from './Quay/QuaysWrapper';
 
 interface MapState {
-  quayStopPointIndexRecord: Record<string, number>;
+  quayStopPointIndexesRecord: Record<string, number[]>;
```

Comment:
Rename

Reply:
I btw checked, both indices and indexes are valid 😄 Thought to keep as indexes so it's faster to recognise